x/epoching/spec/01_state.md

```diff
@@ -0,0 +1,58 @@
+<!--
+order: 1
+-->
+
+# State
+
+## Messages queue
+
+Messages are queued to run at the end of epochs. Queued messages have epoch number to be run and at the end of epochs, it run messages queued for the epoch and execute the message.
```

Comment:
```suggestion
Messages are queued to run at the end of each epoch. Queued messages have an epoch number and for each epoch number, the queues are iterated over and each message is executed.
```

Reply:
Also question, how do order iterating over module queues? Are there cases where this matters?